### PR TITLE
Loosen the requirements to easily install with other packages

### PR DESCRIPTION
**Pull request by shlomikushchi**
as a response to this issue: https://github.com/alpacahq/alpaca-trade-api-python/issues/252

### CODE REVIEW

File: requirements/requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-pandas<0.23
-requests==2.24.0
-urllib3==1.24.3
-websocket-client==0.57.0
-websockets>=8.0
-alpha_vantage==2.2.0
+pandas>=0.18.1
```
Comment (umitanuki) [inline requirements/requirements.txt:1]: I would agree more to loosen other packages. Do we want to support both pandas 0.x and 1.x?

Comment (shlomikushchi) [reply requirements/requirements.txt:1]: we could with:
pandas 
or
pandas>1

and let pylivetrader downgrade it

Comment (umitanuki) [reply requirements/requirements.txt:1]: I'm inclined to do >1 then.

Comment (umitanuki) [reply requirements/requirements.txt:1]: But does it actually work? I think pip will reject such install?

Comment (umitanuki) [reply requirements/requirements.txt:1]: If pylivetrader can't support pandas>1, then maybe we need to do >=0.xx

Comment (shlomikushchi) [reply requirements/requirements.txt:1]: pip will give a warning but will install correctly (for pylivetrader it will remove 1.0.5 and install 0.22.0)

**Pull request merged**